src/widget/wpushbutton.h

```diff
@@ -75,7 +75,6 @@ class WPushButton : public WWidget {
     void focusOutEvent(QFocusEvent* e) override;
     void fillDebugTooltip(QStringList* debug) override;
 
-  protected:
     virtual void restyleAndRepaint();
```

Comment:
revert?

Reply:
no, it's redundant because `protected` is already [in line 70](https://github.com/mixxxdj/mixxx/pull/14179/files#diff-8f2c55e3eeb24c3d39d5a596a249a13bd555e6877488901b74033c04614c7706R70)